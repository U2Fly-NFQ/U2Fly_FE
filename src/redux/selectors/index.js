export * from './auth'
export * from './bookingFlightSelector'
export * from './ticket'
