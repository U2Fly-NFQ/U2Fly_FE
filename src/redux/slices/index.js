export * from './searchSlice'
export * from './filterSlice'
export * from './flightSlice'
