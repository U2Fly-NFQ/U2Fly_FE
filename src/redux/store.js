import { configureStore } from '@reduxjs/toolkit'
import bookingFlightsSlice from './slices/bookingFlightsSlice'

// import loginSlice from './slices/loginSlice'
import flightSlice from './slices/flightSlice'
import airportSlice from './slices/airportSlice'

const store = configureStore({
  reducer: {
    // login: loginSlice.reducer,
    bookingFlight: bookingFlightsSlice.reducer,
    flights: flightSlice.reducer,
    airports: airportSlice.reducer,
  },
})

export default store
