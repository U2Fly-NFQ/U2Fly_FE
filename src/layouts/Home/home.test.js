import { I18nextProvider } from 'react-i18next'
import i18n from '../../translations'
import React, { Suspense } from 'react'
import { store } from '../../redux/store'
import { Provider } from 'react-redux'
import { BrowserRouter as Router } from 'react-router-dom'

import HomeLayout from './'
import { ConfigProvider } from 'antd'
import renderer from 'react-test-renderer'

describe('Home Layout', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <Provider store={store}>
            <ConfigProvider>
              <Router>
                <Suspense>
                  <HomeLayout />
                </Suspense>
              </Router>
            </ConfigProvider>
          </Provider>
        </I18nextProvider>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
