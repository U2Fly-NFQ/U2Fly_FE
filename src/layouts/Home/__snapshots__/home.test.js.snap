// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Layout  renders correctly 1`] = `
<section
  className="ant-layout"
>
  <section
    className="ant-layout"
    style={
      Object {
        "backgroundColor": "#ffffff",
      }
    }
  >
    <header
      className="ant-layout-header home-header"
    >
      <div
        className="sub-nav"
      >
        <nav
          className="grid wide"
        >
          <div
            className="ant-row ant-row-space-between ant-row-middle"
            style={
              Object {
                "marginBottom": -1,
                "marginLeft": -1,
                "marginRight": -1,
                "marginTop": -1,
              }
            }
          >
            <div
              className="ant-col ant-col-xs-0 ant-col-md-12"
              style={
                Object {
                  "paddingBottom": 1,
                  "paddingLeft": 1,
                  "paddingRight": 1,
                  "paddingTop": 1,
                }
              }
            >
              <ul
                className="sub-nav-list"
              >
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    href="#"
                  >
                    028 6681 2733
                  </a>
                </li>
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    href="#"
                  >
                    u2fly@nfq.asia
                  </a>
                </li>
              </ul>
            </div>
            <div
              className="ant-col"
              style={
                Object {
                  "flex": "none",
                  "paddingBottom": 1,
                  "paddingLeft": 1,
                  "paddingRight": 1,
                  "paddingTop": 1,
                }
              }
            >
              <ul
                className="sub-nav-list sub-nav-list-actions"
              >
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    className=""
                    href="/login"
                    onClick={[Function]}
                  >
                    Login
                  </a>
                </li>
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    className=""
                    href="/register"
                    onClick={[Function]}
                  >
                    Register
                  </a>
                </li>
                <li
                  className="sub-nav-list__item"
                >
                  <div
                    className="ant-select ant-select-sm lang-select ant-select-single ant-select-show-arrow"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    styles={
                      Object {
                        "menuPortal": [Function],
                      }
                    }
                  >
                    <div
                      className="ant-select-selector"
                      onClick={[Function]}
                      onMouseDown={[Function]}
                    >
                      <span
                        className="ant-select-selection-search"
                      >
                        <input
                          aria-activedescendant="undefined_list_0"
                          aria-autocomplete="list"
                          aria-controls="undefined_list"
                          aria-haspopup="listbox"
                          aria-owns="undefined_list"
                          autoComplete="off"
                          className="ant-select-selection-search-input"
                          disabled={false}
                          onChange={[Function]}
                          onCompositionEnd={[Function]}
                          onCompositionStart={[Function]}
                          onKeyDown={[Function]}
                          onMouseDown={[Function]}
                          onPaste={[Function]}
                          readOnly={true}
                          role="combobox"
                          style={
                            Object {
                              "opacity": 0,
                            }
                          }
                          type="search"
                          unselectable="on"
                          value=""
                        />
                      </span>
                      <span
                        className="ant-select-selection-item"
                      >
                        <img
                          src="united-kingdom.png"
                          style={
                            Object {
                              "marginBottom": "2px",
                              "marginRight": "4px",
                            }
                          }
                          value="en"
                        />
                        ENG
                      </span>
                    </div>
                    <span
                      aria-hidden={true}
                      className="ant-select-arrow"
                      onMouseDown={[Function]}
                      style={
                        Object {
                          "WebkitUserSelect": "none",
                          "userSelect": "none",
                        }
                      }
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        className="anticon anticon-down"
                        role="img"
                        style={
                          Object {
                            "color": "#fff",
                          }
                        }
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                </li>
              </ul>
            </div>
          </div>
        </nav>
      </div>
      <nav
        className="navbar"
      >
        <div
          className="wide grid"
          style={
            Object {
              "justifyContent": "flex-end",
              "padding": "10px 0",
            }
          }
        >
          <ul
            className="navbar__links"
          >
            <div
              style={Object {}}
            >
              <h3
                className="ant-typography"
                onClick={[Function]}
                style={
                  Object {
                    "WebkitLineClamp": undefined,
                    "alignItems": "center",
                    "color": "#fff",
                    "cursor": "pointer",
                    "display": "flex",
                    "fontSize": "22px",
                    "gap": "0.6rem",
                    "margin": 0,
                    "marginRight": "20px",
                  }
                }
              >
                U2FLy
              </h3>
            </div>
            <li
              className="navbar__link"
            >
              <a
                aria-current="page"
                className="nav-link active"
                href="/"
                onClick={[Function]}
              >
                <p
                  className="navbar__link__data"
                >
                  Home
                   
                </p>
              </a>
            </li>
            <li
              className="navbar__link"
            >
              <a
                className="nav-link"
                href="/flights"
                onClick={[Function]}
              >
                <p
                  className="navbar__link__data"
                >
                  Flights
                   
                </p>
              </a>
            </li>
          </ul>
          <div
            className="navbar_icons"
            onClick={[Function]}
          >
            <div
              className="navbar_icon"
            />
          </div>
        </div>
        <div
          className="navbar__dropdown__modal"
        >
          <ul
            className="navbar__dropdown-menu"
          >
            <li
              className="navbar__dropdown-menu__item"
            >
              <a
                aria-current="page"
                className="active"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li
              className="navbar__dropdown-menu__item"
            >
              <a
                className=""
                href="/flights"
                onClick={[Function]}
              >
                Flight
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </header>
    <main
      className="ant-layout-content"
      style={
        Object {
          "minHeight": "100vh",
        }
      }
    />
    <footer
      className="ant-layout-footer"
    >
      <div
        className="contact-info "
      >
        <div
          className="wide grid"
        >
          <div
            className="footer-info  wide grid"
          >
            <div
              className="footer-info__container "
            >
              <div
                className="ant-row ant-row-space-between"
                style={Object {}}
              >
                <div
                  className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-8 ant-col-lg-6"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__contact"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Need any help?
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__contact-content"
                    >
                      <div
                        className="footer-info__container__contact-content__item"
                      >
                        <h5>
                          CMail to our support team
                        </h5>
                        <h3>
                          u2fly@nfq.asia
                        </h3>
                      </div>
                      <div
                        className="footer-info__container__contact-content__item"
                      >
                        <h5>
                          Call 24/7 for any help
                        </h5>
                        <h3>
                          028 6681 2733
                        </h3>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-3"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Company
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          About Us
                        </li>
                        <li>
                          Testimonials
                        </li>
                        <li>
                          Rewards
                        </li>
                        <li>
                          Work with Us
                        </li>
                        <li>
                          Meet the Team
                        </li>
                        <li>
                          Blog
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-4"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Support
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          Account
                        </li>
                        <li>
                          Faq
                           
                        </li>
                        <li>
                          Legal
                        </li>
                        <li>
                          Contact
                        </li>
                        <li>
                          Affiliate Program
                        </li>
                        <li>
                          Privacy Policy
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-5 ant-col-lg-4"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Other Services
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          Community program
                        </li>
                        <li>
                          Investor Relations
                        </li>
                        <li>
                          Rewards Program
                        </li>
                        <li>
                          Partners
                        </li>
                        <li>
                          PointsPLUS
                        </li>
                        <li>
                          Blog
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-5"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Top cities
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          Ha Noi
                        </li>
                        <li>
                          Da Nang
                        </li>
                        <li>
                          Ho Chi Minh
                        </li>
                        <li>
                          Can Tho
                        </li>
                        <li>
                          Bangkok
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="footer-copyright wide grid"
          >
            <div
              className="footer-copyright"
            >
              <p>
                Copyright © 2022 U2Fly Booking Designed by U2FlyTeam
              </p>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </section>
</section>
`;

exports[`Home Layout renders correctly 1`] = `
<section
  className="ant-layout"
>
  <section
    className="ant-layout"
    style={
      Object {
        "backgroundColor": "#ffffff",
      }
    }
  >
    <header
      className="ant-layout-header home-header"
    >
      <div
        className="sub-nav"
      >
        <nav
          className="grid wide"
        >
          <div
            className="ant-row ant-row-space-between ant-row-middle"
            style={
              Object {
                "marginBottom": -1,
                "marginLeft": -1,
                "marginRight": -1,
                "marginTop": -1,
              }
            }
          >
            <div
              className="ant-col ant-col-xs-0 ant-col-md-12"
              style={
                Object {
                  "paddingBottom": 1,
                  "paddingLeft": 1,
                  "paddingRight": 1,
                  "paddingTop": 1,
                }
              }
            >
              <ul
                className="sub-nav-list"
              >
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    href="#"
                  >
                    028 6681 2733
                  </a>
                </li>
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    href="#"
                  >
                    u2fly@nfq.asia
                  </a>
                </li>
              </ul>
            </div>
            <div
              className="ant-col"
              style={
                Object {
                  "flex": "none",
                  "paddingBottom": 1,
                  "paddingLeft": 1,
                  "paddingRight": 1,
                  "paddingTop": 1,
                }
              }
            >
              <ul
                className="sub-nav-list sub-nav-list-actions"
              >
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    className=""
                    href="/login"
                    onClick={[Function]}
                  >
                    Login
                  </a>
                </li>
                <li
                  className="sub-nav-list__item"
                >
                  <a
                    className=""
                    href="/register"
                    onClick={[Function]}
                  >
                    Register
                  </a>
                </li>
                <li
                  className="sub-nav-list__item"
                >
                  <div
                    className="ant-select ant-select-sm lang-select ant-select-single ant-select-show-arrow"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    styles={
                      Object {
                        "menuPortal": [Function],
                      }
                    }
                  >
                    <div
                      className="ant-select-selector"
                      onClick={[Function]}
                      onMouseDown={[Function]}
                    >
                      <span
                        className="ant-select-selection-search"
                      >
                        <input
                          aria-activedescendant="undefined_list_0"
                          aria-autocomplete="list"
                          aria-controls="undefined_list"
                          aria-haspopup="listbox"
                          aria-owns="undefined_list"
                          autoComplete="off"
                          className="ant-select-selection-search-input"
                          disabled={false}
                          onChange={[Function]}
                          onCompositionEnd={[Function]}
                          onCompositionStart={[Function]}
                          onKeyDown={[Function]}
                          onMouseDown={[Function]}
                          onPaste={[Function]}
                          readOnly={true}
                          role="combobox"
                          style={
                            Object {
                              "opacity": 0,
                            }
                          }
                          type="search"
                          unselectable="on"
                          value=""
                        />
                      </span>
                      <span
                        className="ant-select-selection-item"
                      >
                        <img
                          src="united-kingdom.png"
                          style={
                            Object {
                              "marginBottom": "2px",
                              "marginRight": "4px",
                            }
                          }
                          value="en"
                        />
                        ENG
                      </span>
                    </div>
                    <span
                      aria-hidden={true}
                      className="ant-select-arrow"
                      onMouseDown={[Function]}
                      style={
                        Object {
                          "WebkitUserSelect": "none",
                          "userSelect": "none",
                        }
                      }
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        className="anticon anticon-down"
                        role="img"
                        style={
                          Object {
                            "color": "#fff",
                          }
                        }
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                </li>
              </ul>
            </div>
          </div>
        </nav>
      </div>
      <nav
        className="navbar"
      >
        <div
          className="wide grid"
          style={
            Object {
              "justifyContent": "flex-end",
              "padding": "10px 0",
            }
          }
        >
          <ul
            className="navbar__links"
          >
            <div
              style={Object {}}
            >
              <h3
                className="ant-typography"
                onClick={[Function]}
                style={
                  Object {
                    "WebkitLineClamp": undefined,
                    "alignItems": "center",
                    "color": "#fff",
                    "cursor": "pointer",
                    "display": "flex",
                    "fontSize": "22px",
                    "gap": "0.6rem",
                    "margin": 0,
                    "marginRight": "20px",
                  }
                }
              >
                U2FLy
              </h3>
            </div>
            <li
              className="navbar__link"
            >
              <a
                aria-current="page"
                className="nav-link active"
                href="/"
                onClick={[Function]}
              >
                <p
                  className="navbar__link__data"
                >
                  Home
                   
                </p>
              </a>
            </li>
            <li
              className="navbar__link"
            >
              <a
                className="nav-link"
                href="/flights"
                onClick={[Function]}
              >
                <p
                  className="navbar__link__data"
                >
                  Flights
                   
                </p>
              </a>
            </li>
          </ul>
          <div
            className="navbar_icons"
            onClick={[Function]}
          >
            <div
              className="navbar_icon"
            />
          </div>
        </div>
        <div
          className="navbar__dropdown__modal"
        >
          <ul
            className="navbar__dropdown-menu"
          >
            <li
              className="navbar__dropdown-menu__item"
            >
              <a
                aria-current="page"
                className="active"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li
              className="navbar__dropdown-menu__item"
            >
              <a
                className=""
                href="/flights"
                onClick={[Function]}
              >
                Flight
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </header>
    <main
      className="ant-layout-content"
      style={
        Object {
          "minHeight": "100vh",
        }
      }
    />
    <footer
      className="ant-layout-footer"
    >
      <div
        className="contact-info "
      >
        <div
          className="wide grid"
        >
          <div
            className="footer-info  wide grid"
          >
            <div
              className="footer-info__container "
            >
              <div
                className="ant-row ant-row-space-between"
                style={Object {}}
              >
                <div
                  className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-8 ant-col-lg-6"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__contact"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Need any help?
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__contact-content"
                    >
                      <div
                        className="footer-info__container__contact-content__item"
                      >
                        <h5>
                          CMail to our support team
                        </h5>
                        <h3>
                          u2fly@nfq.asia
                        </h3>
                      </div>
                      <div
                        className="footer-info__container__contact-content__item"
                      >
                        <h5>
                          Call 24/7 for any help
                        </h5>
                        <h3>
                          028 6681 2733
                        </h3>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-3"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Company
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          About Us
                        </li>
                        <li>
                          Testimonials
                        </li>
                        <li>
                          Rewards
                        </li>
                        <li>
                          Work with Us
                        </li>
                        <li>
                          Meet the Team
                        </li>
                        <li>
                          Blog
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-4"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Support
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          Account
                        </li>
                        <li>
                          Faq
                           
                        </li>
                        <li>
                          Legal
                        </li>
                        <li>
                          Contact
                        </li>
                        <li>
                          Affiliate Program
                        </li>
                        <li>
                          Privacy Policy
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-5 ant-col-lg-4"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Other Services
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          Community program
                        </li>
                        <li>
                          Investor Relations
                        </li>
                        <li>
                          Rewards Program
                        </li>
                        <li>
                          Partners
                        </li>
                        <li>
                          PointsPLUS
                        </li>
                        <li>
                          Blog
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
                <div
                  className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-5"
                  style={Object {}}
                >
                  <div
                    className="footer-info__container__company"
                  >
                    <div
                      className="footer-info__container__title"
                    >
                      <h5>
                        Top cities
                      </h5>
                    </div>
                    <div
                      className="footer-info__container__company__content"
                    >
                      <ul>
                        <li>
                          Ha Noi
                        </li>
                        <li>
                          Da Nang
                        </li>
                        <li>
                          Ho Chi Minh
                        </li>
                        <li>
                          Can Tho
                        </li>
                        <li>
                          Bangkok
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="footer-copyright wide grid"
          >
            <div
              className="footer-copyright"
            >
              <p>
                Copyright © 2022 U2Fly Booking Designed by U2FlyTeam
              </p>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </section>
</section>
`;
